module/VuFindApi/src/VuFindApi/Formatter/RecordFormatter.php

```diff
@@ -153,6 +140,20 @@ protected function getRecordPage($record)
         return $urlHelper->getUrl($record);
     }
 
+    /**
+     * Get absolute link to record page
+     *
+     * @param \VuFind\RecordDriver\AbstractBase $record Record driver
+     *
+     * @return string
+     */
+    protected function getRecordPageAbsoluteLink($record)
+    {
+        $recordPage = $this->getRecordPage($record);
+        $serverUrl = $this->serverUrlHelper->getServerUrl($recordPage);
+        return $serverUrl;
```

Comment:
This seems worth simplifying:

```suggestion
        return $this->serverUrlHelper->getServerUrl($recordPage);
```